ui/components/ui/jazzicon/README.mdx

```diff
@@ -0,0 +1,15 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import Jazzicon from './jazzicon.component';
+
+# Jazzicon
+
+Jazzicon component is a wrapper around the jazzicon library to return a React component.
```

Comment:
I think we can be a little more descriptive about what a Jazzicon is here. Always helps to add a link when we use a reference too.

```suggestion
Jazzicon uses the [jazzicon library](https://github.com/MetaMask/jazzicon) to create a graphic identity of an ethereum address
```